[02.01.2025 13:39:34,85] Running norskeutslipp.py 
Loaded .env file from: D:\Scripts\analyse\Telemark\Python\token.env
X_FUNCTIONS_KEY loaded successfully.
Loaded .env file from: D:\Scripts\analyse\Telemark\Python\token.env
GITHUB_TOKEN loaded successfully.
All tasks completed successfully.
<class 'pandas.core.frame.DataFrame'>
RangeIndex: 6294 entries, 0 to 6293
Data columns (total 12 columns):
 #   Column                    Non-Null Count  Dtype 
---  ------                    --------------  ----- 
 0   CO2-ekvivalent (CO2-ekv)  6293 non-null   object
 1   Unnamed: 1                6293 non-null   object
 2   Unnamed: 2                6293 non-null   object
 3   Unnamed: 3                6278 non-null   object
 4   Unnamed: 4                6293 non-null   object
 5   Unnamed: 5                6293 non-null   object
 6   Unnamed: 6                6293 non-null   object
 7   Unnamed: 7                1 non-null      object
 8   Unnamed: 8                1 non-null      object
 9   Unnamed: 9                6293 non-null   object
 10  Unnamed: 10               6293 non-null   object
 11  Unnamed: 11               6282 non-null   object
dtypes: object(12)
memory usage: 590.2+ KB
Attempt 1: Error writing file norskeutslipp_detaljert_telemark.csv. Retrying in 1 seconds...
Attempt 2: Error writing file norskeutslipp_detaljert_telemark.csv. Retrying in 1 seconds...
Error writing file norskeutslipp_detaljert_telemark.csv after 3 attempts: [Errno 13] Permission denied: 'D:\\Scripts\\analyse\\Telemark\\Python\\Temp\\norskeutslipp_detaljert_telemark.csv'
Traceback (most recent call last):
  File "D:\Scripts\analyse\Telemark\Python\Queries\04_Klima_og_energi\Klimagassutslipp\norskeutslipp.py", line 167, in <module>
    is_new_data1 = compare_to_github(
  File "D:\Scripts\analyse\Telemark\Python\Helper_scripts\github_functions.py", line 172, in compare_to_github
    input_df.to_csv(local_file_path, index=False, encoding='utf-8')
  File "C:\Users\_eve1509\AppData\Local\anaconda3\envs\analyse\lib\site-packages\pandas\core\generic.py", line 3772, in to_csv
    return DataFrameRenderer(formatter).to_csv(
  File "C:\Users\_eve1509\AppData\Local\anaconda3\envs\analyse\lib\site-packages\pandas\io\formats\format.py", line 1186, in to_csv
    csv_formatter.save()
  File "C:\Users\_eve1509\AppData\Local\anaconda3\envs\analyse\lib\site-packages\pandas\io\formats\csvs.py", line 240, in save
    with get_handle(
  File "C:\Users\_eve1509\AppData\Local\anaconda3\envs\analyse\lib\site-packages\pandas\io\common.py", line 859, in get_handle
    handle = open(
PermissionError: [Errno 13] Permission denied: 'D:\\Scripts\\analyse\\Telemark\\Python\\Temp\\norskeutslipp_detaljert_telemark.csv'
[02.01.2025 13:40:03,47] Script failed with error code 1 
